python/semantic_kernel/connectors/ai/complete_request_settings.py

```diff
@@ -21,6 +21,7 @@ class CompleteRequestSettings:
     logprobs: int = 0
     token_selection_biases: Dict[int, int] = field(default_factory=dict)
     chat_system_prompt: str = "Assistant is a large language model."
+    function_call: Optional[str] = None
```

Comment:
Is this used for text completion? I believe it's only supported for chat.

Reply:
good one